doc/ChangeLog

```diff
@@ -182,6 +182,7 @@ version 27-git (????-??-??):
   - enhancement: allow to limit visible document types in userpanel documents tab [chilan]
   - enhancement: implemented document approval operator notifications [chilan]
   - enhancement: added '--division' parameter support in lms-notify.php backend script [chilan]
+  - enhancement: new tickets panel in customerinfo form [interduo]
```

Comment:
Rozumiem, że napisałeś to od zera i w ogóle nie korzystałeś z kodu przesłanego przez @mrd83 ?

Reply:
Tak - tak było dla mnie szybciej.

Reply:
@interduo żaden. Na wszelki wypadek zapytałem.